src/js/components/jobs/shareModal.tsx

```diff
@@ -124,22 +131,179 @@ class ShareModal extends React.Component<WrappedProps> {
     return null;
   }
 
+  getScratchOrgInfo() {
+    const { job, scratchOrg } = this.props;
+    if (scratchOrg && job.status === CONSTANTS.STATUS.COMPLETE) {
+      return (
+        <div
+          className="slds-text-longform
+            slds-text-align_center
+            slds-p-bottom_medium"
+        >
+          <p className="slds-text-color_success">
+            {i18n.t('Installation completed successfully.')}
+          </p>
+          <JobMessage job={job} />
+          {job.user_can_edit && (
+            <p>
+              <Trans i18nKey="scratchOrgInfo">
+                To view your org right away, click the “View Scratch Org” button
+                below. To view your org later, copy this link.
+                <br />
+                <strong>
+                  You have also been sent an email to reset the password on your
+                  new org.
+                </strong>
```

Comment:
Remove/hide this sentence for now.

Reply:
I started doing that, but since we're actively working on that feature right now I'd propose leaving this in unless/until we can't deliver the pwd-reset email.

Reply:
@jgerigmeyer ok, I forgot that triggering the password reset was part of the access-org card, so I think that makes sense. It's definitely something we can live without if it turns out to be difficult or not possible though, as long as we have the "View Org" button.